.github/continuous-integration.sh

```diff
@@ -56,7 +60,11 @@ function check-ccache {
 }
 
 function run-build {
-    A2M_CCACHE="1" BUILD_FLAVOUR="Release" A2M_ASSERTS="1" run-step ./build
+    if [[ -z "${PERSIST_DIR}" ]]; then
+        BUILD_FLAVOUR="Release" A2M_ASSERTS="1" run-step ./build
```

Comment:
Duplicating command lines are not a good thing, we should do it like we do it in arc, i.e.

```shell
A2M_CMAKE_DEFS=('BUILD_FLAVOUR="Release"' 'A2M_ASSERTS="1"')

if [[ ! -z "${PERSIST_DIR}" ]]; then
  A2M_CMAKE_DEFS+=('A2M_CCACHE="1"')
fi
...
"${A2M_CMAKE_DEFS[@]}" run-step ./build
```

Reply:
I updated the script now 👍